lib/logger.js

```diff
@@ -1,5 +1,5 @@
 import { getLogger } from 'appium-logger';
 
-let logger = getLogger('iosCrashLog');
+let logger = getLogger('iOS');
```

Comment:
But this is the `appium-ios-driver` package. I'm not sure how `iosCrashLog` came to be in there. I would be happy to break this out into its own package, but I'm not sure if we use it for anything else (though, come to think of it, it could be useful for some other packages, like `appium-ios-simulator`, and `appium-uiauto`, to have access to the logs).